src/main/java/org/ambraproject/wombat/service/remote/AbstractContentApi.java

```diff
@@ -128,7 +128,7 @@ private URI buildUri(ContentKey key, RequestMode mode) throws IOException {
     RepoConfig repoConfig = getRepoConfig();
     UrlParamBuilder requestParams = UrlParamBuilder.params();
     key.setParameters(requestParams);
-    String repoBucketName = repoConfig.bucketName;
+    String repoBucketName = key.getBucketName() != null ? key.getBucketName() : repoConfig.bucketName;
```

Comment:
The change in rhino guarantees that a bucket name will be included with the article metadata, correct? So please remove the repoConfig.bucketName code throughout this class.